src/editor.py

```diff
@@ -418,7 +418,8 @@ def generate_scoring_plays(self,files):
             return scoringplays
 
 
-    def generate_highlights(self,files):
+    def generate_highlights(self,files,myteam="",theater_link=False):
+        import datetime

```

Comment:
Good catch. I was getting an error before I imported datetime, but that's because you imported it differently than I was coding for. I'm removing this extra import and fixing my code now.